.github/workflows/PR-build.yml

```diff
@@ -258,6 +262,19 @@ jobs:
         if: ${{ needs.changes.outputs.changed == 'true' }}
         run: cp -R build aws-otel-collector/build
 
+      - name: Configure AWS Credentials
```

Comment:
Same here, can you break this in two prs. I assume this is a backport of what is on main. So can you break this in two PRs: one for the changes that are unrelated to the work that you are doing and another for the functionality that you are adding/changing.